src/garage/plotter/plotter.py

```diff
@@ -5,13 +5,19 @@
 from multiprocessing import JoinableQueue, Process
 import platform
 from threading import Thread
+import contextlib
+
+import tensorflow as tf
```

Comment:
We can't assume that tf is actually installed when running this file, so we should only run this import once `tf_plot` has been set to True (this requires ignoring pylint).